src/appState.ts

```diff
@@ -45,6 +45,8 @@ export function getDefaultAppState(): AppState {
     collaborators: new Map(),
     shouldCacheIgnoreZoom: false,
     showShortcutsDialog: false,
```

Comment:
We should also rename this to shortcutsDialogOpened, but that's for another PR

Reply:
`shortcutsDialogOpened` sounds more like a user action whereas `showShortcutsDialog` sounds more like a variable that tells whether the element should hide or show on DOM. 
So  `showShortcutsDialog` looks better to me as its a state.
We can may be rename it better to `renderShortcutsDialog` | `shouldRenderShortcutsDialog` ?

Reply:
`show*` suggests a transient action, so I'd go with `shortcutsDialogOpened`, too (also, it should be `shortcutsDialogOpen` without the `ed`, to be grammatically correct) -- but like @vjeux said, that's different PR.